packages/connection-storage/src/renderer.ts

```diff
@@ -0,0 +1,3 @@
+export { ConnectionStorageRenderer as ConnectionStorage } from './connection-storage';
```

Comment:
If you are exporting them from one file, you will make it so that webpack will pull the code for both on import (in theory tree shaking helps with this sometimes, but it's hard to configure right and validate, I'd not rely on this), it would make more sense for these classes to literally be in separate files